code/game/objects/machinery/tesla_turret.dm

```diff
@@ -0,0 +1,190 @@
+/obj/item/tesla_turret
+	name = "tesla turret"
+	desc = "A turret that drains plasma of nearby xenomorphs."
+	icon = 'icons/obj/machines/deployable/sentry/tesla.dmi'
+	icon_state = "grounding_rod_open0"
+	max_integrity = 250
+	var/obj/item/cell/battery
+
+/obj/item/tesla_turret/Initialize(mapload)
+	. = ..()
+	var/matrix/M = transform
+	M.Scale(0.75)
+	transform = M
```

Comment:
Whut? why?

Reply:
To make it extremely clear that this is undeployed and an item due to lack of its unique sprites.